docs/source/markdown/podman-play-kube.1.md

```diff
@@ -36,6 +36,10 @@ The [username[:password]] to use to authenticate with the registry if required.
 If one or both values are not supplied, a command line prompt will appear and the
 value can be entered.  The password is entered without echo.
 
+**--network**=*cni networks*
+
+A comma-separated list of the names of CNI networks the pod should join.
+
```

Comment:
an example of valid input might be nice in the example section.

Reply:
Done